core/vdbe/mod.rs

```diff
@@ -2049,11 +1985,12 @@ impl Program {
                     target_pc,
                 } => {
                     let cursor = cursors.get_mut(cursor).unwrap();
-                    match cursor.exists(&state.registers[*rowid_reg])? {
-                        CursorResult::Ok(true) => state.pc += 1,
-                        CursorResult::Ok(false) => state.pc = *target_pc,
-                        CursorResult::IO => return Ok(StepResult::IO),
-                    };
+                    let exists = io!(cursor.exists(&state.registers[*rowid_reg]));
```

Comment:
so we don't have NotExistsAsync/NotExistsAwait to be consistent with other instructions? (this is not a change request just a thought)